src/WTG.Analyzers.Test/TestData/VisibilityAnalyzer/Partial/Source.cs

```diff
@@ -3,11 +3,21 @@ namespace ns
 	partial class Foo
 	{
 		private partial int Bar();
+		public partial void FooBar();
+		private partial int FooBarBaz(out int value);
+
+		private partial void Qux();
+		private partial void Quux(out int value);
 	}
 
 	partial class Foo
 	{
 		private partial int Bar() { return default; }
+		public partial void FooBar() { }
+		private partial int FooBarBaz(out int value) { value = default; return default; }
```

Comment:
Another option for methods that we don't care about:

```suggestion
		private partial int FooBarBaz(out int value) => throw null;
```

Reply:
I'll apply this.